Spec/Tests/UtilsTests.swift

```diff
@@ -0,0 +1,42 @@
+import Ably
+import Nimble
+import XCTest
+
+class UtilsTests: XCTestCase {
+    
+    struct Constants {
+        static let backoffValues = [ 1.0, 1.3333333333333333, 1.6666666666666667, 2.0, 2.0 ]
```

Comment:
Feel free to ignore because I don't know if this is a standard thing in Objective-C, but if it's possible to use computed values here (ie `4/3` rather than `1.3333333333333333`) that would look more readable IMO.

Reply:
https://github.com/ably/ably-cocoa/pull/1520/commits/12b1cdc9c48990cc0d35cb01947480a4b212d8c7